package.json

```diff
@@ -113,7 +113,7 @@
     "classlist.js": "1.1.20150312",
     "core-js": "3.7.0",
     "focus-trap": "6.5.0",
-    "fundamental-styles": "0.18.1-rc.25",
+    "fundamental-styles": "0.19.0-rc.23",
```

Comment:
Actually, are we sure we want to bring that version without adopting the changes?

Reply:
We need a newer version with fixed selectors, otherwise unit tests will fail while searching for the invalid element